vault/identity_store_oidc_provider.go

```diff
@@ -881,20 +881,20 @@ func (i *IdentityStore) pathOIDCCreateUpdateClient(ctx context.Context, req *log
 
 	if client.ClientID == "" {
 		// generate client_id
-		clientID, err := base62.Random(32)
+		clientID, err := base62.Random(29)
```

Comment:
Nit: should these be constants?

Reply:
I think so too. It'll probably also make it explicitly clear as to how we decided on the length of the random strings by defining the constant as `maxLength - len("id_")` or so